### PR TITLE
add_Urdu

### PR DESCRIPTION
**Pull request by yethrosh**
I have added necessary configuration of Urdu Wikipedia. Please merge it, Thanks.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -713,6 +713,24 @@
...
+        database = 'urwiki_p',
+        
+        wikipedia_domain = 'ur.wikipedia.org',
+        citation_needed_category = 'ماخذ_میں_نامکمل_اندراجات',
+        beginners_link = 'https://ur.m.wikipedia.org/wiki/ویکیپیڈیا:حوالہ_دہی',
+        beginners_link_title = 'ویکیپیڈیا:حوالہ دہی',
+        lead_section_policy_link = '',
+        lead_section_policy_link_title = '',
+        citation_needed_templates = [
+            'حوالہ_درکار',
```
Comment (eggpi) [inline config.py]: Please use a space rather than an underscore: https://travis-ci.org/eggpi/citationhunt/builds/463840146#L554

Comment (yethrosh) [reply config.py]: Glad to hear you. Please modify this patch as you wish, Thanks. :)

**Comment by eggpi** [comment]
Merged as 32b66aaeb1e078c455eb00ede8401d4717086710. Thanks!

**Comment by yethrosh** [comment]
Tool throws error:

https://tools.wmflabs.org/citationhunt/ur

**Comment by eggpi** [comment]
Hm, the error seems to be because we don't have any snippets to serve.

The [category we're using](https://ur.wikipedia.org/wiki/زمرہ:ماخذ_میں_نامکمل_اندراجات) only has 3 articles, which is way too little. However, the template we're trying to index has [1185 transclusions](https://tools.wmflabs.org/templatecount/index.php?lang=ur&namespace=10&name=%D8%AD%D9%88%D8%A7%D9%84%DB%81+%D8%AF%D8%B1%DA%A9%D8%A7%D8%B1).

Is there a different category that tracks the transclusions of that template? Or is maybe something wrong with the category, since it doesn't reflect all transclusions of the template?

**Comment by yethrosh** [comment]
Now you can see 1100+ entries in this category:

https://ur.wikipedia.org/wiki/زمرہ:ماخذ_میں_نامکمل_اندراجات

**Comment by eggpi** [comment]
Cool, thanks! Looks like now we're able to extract ~350 snippets, so it seems like we're fine.

I should point out that I'm clicking through some articles in that category and a lot of them just look pretty empty, and I don't even see where exactly we're including the template. Examples:

-https://ur.wikipedia.org/wiki/2014_%D8%A7%D8%B3%D8%B1%D8%A7%D8%A6%DB%8C%D9%84_%D9%88_%D8%BA%D8%B2%DB%81_%D8%AA%D9%86%D8%A7%D8%B2%D8%B9
- https://ur.wikipedia.org/wiki/%D8%A7%D9%84%DB%8C%D8%B3%D8%AA%D8%A7

but then again I might be missing something, as I don't really know how to read Urdu.

Anyway, I just wanted to point that out in case it indicates other problems with the category, but it looks like Citation Hunt is working fine now.

**Comment by yethrosh** [comment]
Thanks, now it works fine :)

**Pull request closed without merging**